playwright.config.ts

```diff
@@ -7,14 +7,15 @@ const config: PlaywrightTestConfig = {
   testDir: "./e2e/specs",
   timeout: 3 * 60 * 1000,
   expect: {
-    timeout: 20 * 1000,
+    timeout: 80 * 1000,
   },
   fullyParallel: true,
+  outputDir: "../test-results/results",
   forbidOnly: !!process.env.CI,
   retries: 0,
   reporter: process.env.CI
     ? [["junit", { outputFile: "results.xml" }], ["html"]]
-    : [["html"]],
+    : [["html", { outputFolder: "../test-results/report" }]],
```

Comment:
Until the webpack watching issue is resolve we can use this.

Reply:
Awesome. We need to edit the action file too. Also update line 16 too.

Reply:
github action?